README.md

```diff
@@ -320,6 +322,23 @@ if erl_triggered // quota left in the quotaKey bucket
 if !erl_triggered // ERL wasn't triggered in this call, so we haven't identified the remaining quota.
 ```
 
+### Use of fixed window on Take and TakeElevated
+If you want to use fixed window on Take or TakeElevated, you can do so by setting the `fixed_window` property in the bucket configuration to `true` (default `false`). This will refill the bucket at the specified interval instead of granular.
+
+On top of that, you can use the `fixed_window` property in the `configOverride` parameter to safely activate/deactivate the new fixed_window algorithm from the client on-demand.
+
+The following table describes how the fixed window bucket configuration and the fixed window param interact to activate the fixed window algorithm:
+
+| fixed_window bucket config | fixed_window param | Fixed Window Enabled |
+|----------------------------|--------------------|----------------------|
+| true                       | true               | Yes                  |
+| true                       | false              | No                   |
+| true                       | not provided       | Yes                  |
+| false                      | true               | No                   |
```

Comment:
I'm not sure what you mean here.

Reply:
If I understood correctly, you can do `limitd.take(...., { fixed_window: true})` and this should enable the fixed window behaviour. So if bucket is configured for sliding window but you pass this, it should get fixed_window enabled. So the last column on that table row should be `Yes` (false, true => yes).

Or did I misunderstood something?

Reply:
No. Maybe it needs further clarification.
This is an AND gate. The flag not being provided is treated as "true".

It's only activated if both are true. The benefit of this is:
- We configure the buckets as we intend to use them (for instance bucket A using fixed window, bucket B using sliding).
- During the adaptation time we can have a feature flag being passed in the fixed_window parameter just in case we need to rollback to sliding window (current behavior). 
- When we're ready to test it in production we set the flag to true, and only those buckets configured for fixed window will be moved to fixed window. The remaining will keep using sliding window.
- Whenever we're ready we remove the feature flag, hence stop sending the fixed_window parameter, we will just rely on the configuration.

Reply:
I would say this indeed requires more clarification, because from the paragraph I quoted

> On top of that, you can use the `fixed_window` property in the `configOverride` parameter to safely activate/deactivate the new fixed_window algorithm from the client on-demand. This parameter acts as a feature flag to safely deploy the change to produciton.

To me it reads like "if the config is disabled at the bucket level, you can still activate it via the config object you pass to the request on demand.

Reply:
fixed